draft-ietf-tls-ctls.md

```diff
@@ -348,6 +390,11 @@ representing the uncompressed values.  For example:
 }
 ~~~~~
 
+Certificates are a major contributor to the size of a TLS handshake.  In order
+to avoid this overhead when the parties to a handshake have already exchanged
+certificates, a compression profile can specify a dictionary of "known
+certificates" that effectively acts as a compression dictionary on certificates.
```

Comment:
I'm concerned about the implications of not reconstructing the transcript in this case.

Reply:
I'm reasonably confident that this is formally safe.  Interpretation of compressed certificates is driven exclusively by the template, which is in the transcript.

This arrangement is potentially more error-prone, as transcript->logical layer conversion is not verified by the handshake.

Reply:
I'm persuaded.